src/database/sequlize-handler/migrations/08_tests__favorite.js

```diff
@@ -0,0 +1,15 @@
+const Sequelize = require('sequelize');
```

Comment:
not sure you meant to create this migration file for `tests` table and add an `is_favorite` column?